scst-store/sca-scans/sca-scanning-results.md

```diff
@@ -1,4 +1,52 @@
 # SCA scanning results
+## <a id='110'></a>1.1.2
+
+### <a id='black-duck-ba'></a>Black Duck Binary Analysis (BDBA)
+
+#### <a id='api-backend'></a>API backend
+
+* Date: March 25, 2022
+* Results: No known vulnerabilities
+
+#### <a id='cli-sr'></a>CLI
+
+* Date: March 25, 2022
+* Results: No known vulnerabilities
+
+### <a id='grype-sr'></a>Grype
+
+Version: 0.33.1
+
+#### <a id='api-backend-ci'></a>API Backend Container Image
+
+* Date: March 25, 2022
+* Results: No critical, and no high vulnerabilities. One low vulnerabilities. For more information, see the [API Backend Container Image v1.1.2 CycloneDX file content](api-backend-container-image-v1.1.2-grype-result.md).
```

Comment:
Maybe reword to: "No critical, high and medium vulnerabilities."

Just bringing this up because we mention medium vulnerabilities in the other sections.